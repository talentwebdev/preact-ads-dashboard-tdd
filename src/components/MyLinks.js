import React from "react";
import { Link } from "react-router-dom";
function MyLinks() {
  return (
    <div className="myLinks">
      <div id="title">
        <p>My Links</p>
      </div>
      <div id="body">
        <Link to="/">BHHS Tours</Link>
        <Link to="/">Training Videos</Link>
        <Link to="/">Marketing Materials</Link>
      </div>
    </div>
  );
}

export default MyLinks;
