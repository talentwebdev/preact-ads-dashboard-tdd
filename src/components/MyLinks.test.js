import { shallow } from "enzyme";
import MyLinks from "./MyLinks";

describe("MyLinks", () => {
  let myLinksContainer;
  beforeEach(() => (myLinksContainer = shallow(<MyLinks />)));
  test("should render the div element", () => {
    expect(myLinksContainer.find("div")).toBeTruthy();
  });
  it("renders MyLinks correctly", () => {
    expect(myLinksContainer).toMatchSnapshot();
  });
});
