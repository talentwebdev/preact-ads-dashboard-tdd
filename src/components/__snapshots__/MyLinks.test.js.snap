// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyLinks renders MyLinks correctly 1`] = `ShallowWrapper {}`;
